docs/2022/ui/updates/soham/2022-06-24.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Week 1
+author: Soham Banerjee
+author_url: https://github.com/soham4abc
+author_image_url: https://avatars.githubusercontent.com/u/63705023?v=4
+tags: [gsoc22, react, API]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2022 Soham Banerjee <sohambanerjee4abc@hotmail.com>
+-->
+
+## All Recent Jobs Page
+
+_(June 16th, 2022)_
+
+Implementation of the All Recent Jobs page in the UI started.
+The response from the `/jobs` endpoint was used in order to get the data from the backend server
+
+Response format of the API: <br/>
+![Screenshot from 2022-06-24 10-04-12](../../../../../static/img/docs_img/docs_api_res.png)
```

Comment:
```suggestion
![Screenshot from 2022-06-24 10-04-12](/img/docs_img/docs_api_res.png)
```

And please move images to a folder under `static/img/reactUI`for consistency

Reply:
Resolved